Project.toml

```diff
@@ -3,5 +3,11 @@ uuid = "7ce85f02-24a8-4d69-a3f0-14b5daa7d30c"
 authors = ["bill-becker <William.Becker@nrel.gov>"]
 version = "0.1.0"
 
+[deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+
 [compat]
-julia = "1.7"
\ No newline at end of file
+julia = "1.7"
```

Comment:
You can still update the package environment to julia 1.8, and then please add/append 1.8 to the julia compat versions, i.e. julia = "1.7, 1.8"

Reply:
@Bill-Becker I removed the packages from the main environment. Just wanted to get your eyes on it one last time before I merged - wasn't too sure what `project_hash` is and wanted to make sure I'm not updating anything unexpectedly!